airbyte-webapp/src/components/EntityTable/utils.tsx

```diff
@@ -73,43 +73,32 @@ export function getEntityTableData<
 }
 
 export const getConnectionTableData = (
-  connections: WebBackendConnectionRead[],
-  sourceDefinitions: SourceDefinitionRead[],
-  destinationDefinitions: DestinationDefinitionRead[],
+  connections: WebBackendConnectionListItem[],
```

Comment:
:information_source: This no longer requires the source/destination definitions since we also made `icon` to be part of the response in the connections/list API in this PR.